packages/mdc-tab-scroller/mdc-tab-scroller.scss

```diff
@@ -59,6 +59,18 @@
   will-change: transform;
 }
 
+.mdc-tab-scroller--start-aligned .mdc-tab-scroller__scroll-content {
```

Comment:
The docs were updated but these classes were not

Reply:
lololol i'm a fool 😮